### ISSUE TITLE
Any DataProvider for both production and debugging use?

### ISSUE THREAD

**Issue by hydroper** [unicode-org/icu4x#3180]
Constructors like `Collator` require a data provider. Is there one I can use everywhere (not the test one)?

**Comment by sffc** [comment]
Yes; please read the tutorial and let us know if there are any unanswered questions:

https://github.com/unicode-org/icu4x/blob/main/docs/tutorials/data_management.md

**Comment by hydroper** [comment]
@sffc I'll read it! But is there a chance a crate like Fluent is shipping the same data?

**Comment by sffc** [comment]
@zbraniecki can address questions about Fluent's use of ICU4X

**Comment by hydroper** [comment]
I opened one issue there: https://github.com/projectfluent/fluent-rs/issues/314#issuecomment-1460734898 I've replied to him about where I believe Fluent may use such data.
